src/components/Breadcrumb/Breadcrumb.jsx

```diff
@@ -1,12 +1,12 @@
 import React, {Fragment} from 'react';
 import PropTypes from 'prop-types';
-import styles from './Breadcrumb.scss';
+import './Breadcrumb.scss';
 import classnames from 'clsx';
 import {ChevronRight} from '~/icons';
 
 export const Breadcrumb = ({children, className, ...props}) => {
     const classNames = classnames(
-        styles.breadcrumb,
+        'breadcrumb',
```

Comment:
where does this class comes from ?

Reply:
Good question 🙂 I saw it in the original jsx file so I left it in but I've removed it now.